setup.cfg

```diff
@@ -32,4 +32,12 @@ install_requires =
 [flake8]
 max-line-length = 100
 doctests = True
-exclude = pyarn/parsetab.py
+exclude = venv,.tox,pyarn/parsetab.py,pyarn/parser.out
+show-source = True
+
+[isort]
+profile = black
+line_length = 100
+
+[black]
+line_length = 100
```

Comment:
Does black really read setup.cfg? I feel there should have been formatting changes after bumping the line length, but there were none

Reply:
Looks like black uses pyproject.toml: https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#black-compatible-configurations

Reply:
The default for black is 88, put it there just to be sure :D

Reply:
Aren't we using line length 100 everywhere else though? When I add
```
[tool.black]
line-length = 100
```
to pyproject.toml, I see black making a few formatting changes